app/src/androidTest/java/org/simple/clinic/di/TestAppComponent.kt

```diff
@@ -11,6 +11,7 @@ import org.simple.clinic.bloodsugar.BloodSugarRepositoryAndroidTest
 import org.simple.clinic.bp.BloodPressureHistoryListItemDataSourceTest
```

Comment:
Can we please reword the commit message to follow the convention of "If applied, this commit will <commit message>"?